pymc/distributions/continuous.py

```diff
@@ -573,9 +574,13 @@ def logcdf(value, mu, sigma):
 
         Parameters
         ----------
-        value: numeric or np.ndarray or `TensorVariable`
+        value : numeric or ndarray or TensorVariable
             Value(s) for which log CDF is calculated. If the log CDF for multiple
             values are desired the values must be provided in a numpy array or `TensorVariable`.
+        mu : float
+            Mean. Defaults to 0.
+        sigma : float
+            Standard deviation (sigma > 0).
```

Comment:
I think `mu` and `sigma` parameters can be numpy array/tensor variables as well.

Reply:
I think we only mention the base case

Reply:
numpy uses float or array_like of floats: https://numpy.org/doc/stable/reference/random/generated/numpy.random.normal.html

Reply:
I have a similar comment below, missed this one. I'd propose something like `float, array_like or TensorVariable` not sure if we need to add `of float` to both array and tensor types :thinking:

Reply:
`float, array_like or TensorVariable` looks good to me. Should I update the pull request with this?

Reply:
Use `tensor_like of float`. We will update the glossary to have a definition for the type like numpy does with array_like.